geolocator_platform_interface/lib/src/geolocator_platform_interface.dart

```diff
@@ -159,6 +159,21 @@ abstract class GeolocatorPlatform extends PlatformInterface {
     throw UnimplementedError('getPositionStream() has not been implemented.');
   }
 
+  /// Asks the user for Temporary Precise location access (iOS 14 or above).
+  ///
+  /// Throws a [AccuracyDictionaryNotFoundException] when the key
```

Comment:
Should be update to:

```suggestion
  /// Throws a [PermissionDefinitionsNotFoundException] when the key
```